data/endf8.py

```diff
@@ -39,16 +39,26 @@
 h5.attrs["last-resonance-group"] = fdy.get_default_group_structure().last_res_grp
 h5.attrs["library"] = lib_name
 
+# Define a helper function to process nuclides
+def process_nuclide(name: str, endf_filename: str, dilutions: list[float] | None = None):
+    N = fdy.FrendyMG()
+    N.name = name
+    N.endf_file = base + endf_filename
+    N.label = N.name + " from ENDF/B-8.0"
+    N.temps = temps
+    if dilutions:
+        N.dilutions = dilutions
+    N.process(h5)
+    return N
```

Comment:
Maybe we could extract this function and put it in the `tools\frendy.py` module ? That would allow us to re-use it for future library generation scripts. In that case, we could remove the " from ENDF/B-8.0" portion of the string.

I am also thinking about how we could parallelize the library generation in the future. This would require processing each isotope in a separate directory, because the intermediate processing files used by FRENDY have the same name.